packages/backend/server/src/__tests__/models/feature-user.spec.ts

```diff
@@ -123,3 +123,21 @@ test('should not switch user quota if the new quota is the same as the current o
 
   t.not(quota?.reason, 'test not switch');
 });
+
+test('should use pro plan as free for selfhost instance', async t => {
+  // @ts-expect-error override
+  AFFiNE.isSelfhosted = true;
```

Comment:
should reset it after test?

Reply:
ava tests are running in isolated sandboxes